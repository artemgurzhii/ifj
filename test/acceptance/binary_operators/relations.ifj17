Scope
dim a as integer
dim b as integer

a = 5
b = 1

if a > 1 then
  b = b + 1
end if
End Scope
