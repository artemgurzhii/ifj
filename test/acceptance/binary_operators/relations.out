.IFJcode17
JUMP Scope
LABEL Scope
DEFVAR GF@a
DEFVAR GF@b
MOVE GF@a int@5
MOVE GF@b int@1
CREATEFRAME
DEFVAR TF@temp_bool
GT TF@temp_bool GF@a int@1
JUMPIFEQ RES_IF_1 TF@temp_bool bool@true
JUMP END_IF_1
LABEL RES_IF_1
ADD GF@b GF@b int@1
JUMP END_IF_1
LABEL END_IF_1
